app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,84 @@
+body {
```

Comment:
Ten plik jest niepotrzebny. Można go spokojnie usunąć, bo dodawanie styli nie było częścią zadania.

Reply:
I've used 'scaffold' in the first version of creating customer model and after 'destroy scaffold' some changes have remained